recaf-assembler/src/main/java/me/coley/recaf/assemble/ast/HandleInfo.java

```diff
@@ -89,9 +90,9 @@ public String getDesc() {
 	@Override
 	public String print() {
 		if (isMethod) {
-			return tag + " " + owner + '.' + name + desc;
+			return tag + " " + EscapeUtil.escapeSpace(owner + '.' + name) + desc;
 		} else {
-			return tag + " " + owner + '.' + name + ' ' + desc;
+			return tag + " " + EscapeUtil.escapeSpace(owner + '.' + name) + ' ' + desc;
```

Comment:
Do we want to also escape the descriptor?
If the owner should escape and the desc returns its owner as the type, we logically should escape it too.

Reply:
yes i have forgotten, i am going to change that